### PR TITLE
Add bea changes

### PR DESCRIPTION
**Pull request by ynion-mabeamae**
Please review that I attached in my module.

### CODE REVIEW

File: pack/ynion.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+def display_menu():
+    while True:
+        print("\nHello! I' Ma. Bea Mae Ynion")
```
Comment (baexoxyn) [inline pack/ynion.py:3]: Check your missing typing.

File: pack/ynion.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        print("4. Exit")
+        
+        choice = int(input("Enter your choice:"))
+
+        match choice:
+            case 1:
+                print("\nDate of Birth : July 29, 2004")
+                print("\nAge : 20")
+                print("\nEmail : mabeamaeynion@gmail.com")
+                print("\nReligion : Roman Cathokic")
```
Comment (baexoxyn) [inline pack/ynion.py:16]: Check your typing in Religion section.

**Review by baexoxyn — CHANGES_REQUESTED** [review]
Please check your code before I approve the changes.

**Review by ynion-mabeamae — COMMENTED** [review]
Great.

**Pull request merged**